src/docs/patterns/internationalization.mdx

```diff
@@ -52,7 +53,22 @@ When showing a list of languages, the language names should be presented in thei
 
 ## Converting Units, Currencies, and Formats
 
-Units, currency, and date / time formats are usually linked with a particular locale. However, there can be differences in usage even within a single locale. You should provide users with the option to change these values (if appropriate for your application) in their locale settings.
+Units, currency, and formats are usually linked with a particular locale. However, there can be differences in usage even within a single locale. You should provide users with the option to change these values (if appropriate for your application) in their locale settings.
+
+Some commonly influenced formats include:
+
+-   Phone number
+-   Names (given name first or last)
+-   Address (street name first or country name first)
+-   Date and time
+-   Numbers (comma vs. period)
+-   Punctuations
```

Comment:
```suggestion
-   Phone number
-   Names (e.g., given name vs. surname order)
-   Address (e.g., street name first or country name first)
-   Date and time
-   Numbers (comma vs. period)
-   Punctuation
```
any examples of where punctuation differs? I know quotations are different, but was there something else you had in mind?

Reply:
Mostly for that quotation thing«»  but… there can be a lot of Asian things that differ from those Latin based languages

。、《》「」